buildkite/bazelci.py

```diff
@@ -397,6 +397,10 @@
 
 SKIP_TASKS_ENV_VAR = "CI_SKIP_TASKS"
 
+VALIDATE_CONFIG_OPTION = "validate_config"
```

Comment:
I'm not a fan of extracting strings used only once into global constants... can we just put them where they are actually used?

Reply:
Done